src/Tribe/Aggregator/Record/Queue_Realtime.php

```diff
@@ -110,35 +111,48 @@ public function render_update_message() {
 	}
 
 	/**
-	 * Handle queue ajax requests
+     * Action to reply every time a heart beat is executed to send the progress of EA if an EA record is present.
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: a7e50cbeb69cd719d001e5786874ac6b</sub>